tools/autograd/derivatives.yaml

```diff
@@ -2959,3 +2959,6 @@
 - name: _reshape_copy(Tensor self, SymInt[] size) -> Tensor
```

Comment:
need to fix thirdparty accidental changes

Reply:
This is not an accidental change but YAML being YAML. The line starts with `-`.
Had I deleted it it would show up as a double minus sign `- - name: _reshape_copy(Tensor self, SymInt[] size) -> Tensor`.